BLSQ/iaso
# POLIO-1211 add Authorization start date

New field on VaccineAuthorization model.

Related JIRA tickets : POLIO-1211

## Self proofreading checklist


## Changes

Add a New field starting date to vaccine authorization

## How to test

Check the tables in vacc auth and details, the field should be there. Create/edit n authorisation and the new required field must be in the modal

## Print screen / video

https://github.com/BLSQ/iaso/assets/45785235/cd0df7de-6648-4d7d-8c5f-d3b8684b2e0a
